config.toml

```diff
@@ -113,6 +113,10 @@ lastmod = ["lastmod", ":fileModTime", ":default"]
 [params.docs_menu]
   title = "PAKETO DOCS"
 
+[params.docs_search]
+  api_key = "bf2d426728685665ceaf22fac947f10f"
```

Comment:
Should this API key be exposed like this?

Reply:
@ryanmoran, correct me if I'm wrong, but my understanding is that's the *public* search API key. Safe to be checked in. All it lets you do is query our search backend.

Reply:
Yes. This is not a secret and so can be checked in.